lib/bridge/IrcHandler.js

```diff
@@ -696,6 +712,7 @@ IrcHandler.prototype.onPart = Promise.coroutine(function*(req, server, leavingUs
 
 IrcHandler.prototype.onMode = Promise.coroutine(function*(req, server, channel, by,
                                                 mode, enabled, arg) {
+    this._callCountMetrics["mode"]++;
```

Comment:
for safety reasons, I think it might be best to use a function instead. It'd be dead simple, but probably worth it given the fairly wide range of possible values.

```
function _incrementCallCount(metric) {
  if (!this._callCountMetrics[metric]) this._callCountMetrics[metric] = 0;
  this._callCountMetrics[metric]++;
}
```

Reply:
Safety for what? Is it possible for any of these properties to be undefined?

Reply:
Developer safety. At some point someone will forget to add the default to the object and be met with angry stack traces. Might as well mitigate it while we can.

Reply:
fiiine